### ISSUE TITLE
 Add "Home"-Button to entry-editor 

### ISSUE THREAD

**Issue by JVZavatin** [alescrocaro/jabref#3]
When an entry is picked from the table of entries the entry-editor is positioned to this entry.
When a group is selected, NOT containing this entry, the entry-editor keeps his position/content (what else...),
there is NO highlighting in the table of entries any more (what else...).

When a (super-)group is highlighted, again containing the entry of the entry-editor,
there is still NO highlighting in the table of entries. This should be improved, it is contra-intuitive
The corresponding entry in the table of entries should be highlighted again.

A new button "Home" should be added to the in the entry-editor.
It should force the selection of groups to walk the tree backwards until the
group is containing this entry and the entry should be highlighted.
Alternatively, the group should be switched to "All entries",
in case walking the tree backwards is to costly.

This button allows for jumping back from a "search" in the groups, i.e a tentative
switching between groups.

Best, HSvia

Issue original: https://github.com/JabRef/jabref/issues/9108

**Comment by JVZavatin** [comment]
Not sure if this specific button is really necessary.

    When an entry is picked from the table of entries the entry-editor is positioned to this entry. heavy_check_mark

Good

    When a group is selected, NOT containing this entry, the entry-editor keeps his position/content heavy_check_mark

Good

    there is NO highlighting in the table of entries any more [when having selected a group NOT containing this entry] heavy_check_mark

Good

    When a (super-)group is highlighted, again containing the entry of the entry-editor,
    there is still NO highlighting in the table of entries. x

Agree, there should be highlighting again.

In short: Whenever the entry, which is currently open in the entry editor is in the group, it should be highlighted.

If the highlighting were to work properly, clicking on the group "all entries" or alternatively a toggle for "show entries not in this group" (refs https://github.com/JabRef/jabref/issues/9073), would provide the same outcome as implementing a "home" button - except for "jumping" to the selected entry. Workaround for the jumping would be to use the search bar and search for the entry.
